api/v1/sriovoperatorconfig_types.go

```diff
@@ -22,6 +22,8 @@ type SriovOperatorConfigSpec struct {
 	LogLevel int `json:"logLevel,omitempty"`
 	// Flag to disable nodes drain during debugging
 	DisableDrain bool `json:"disableDrain,omitempty"`
+	// Flag to enable OVN hardware offload
+	EnableOvnOffload bool `json:"enableOvnOffload,omitempty"`
```

Comment:
Use OVS instead of OVN? I felt sometimes OVN offload is referring to smart NIC offload.
Also does it make sense to not expose the boolean flag ?
for example, if switchdev poilicy is configured on node interface with default route, then render the MCO manifests.
If switchdev policy is configured on non-default interface (switchdev CNI in the future), then don't render MCO manifests.

Reply:
OVS offload could imply the openshift-sdn can also be offloaded.

If we use the nodepolicy to control the MC and MCP, it would be difficult to synchronize the reboot logic between MCO and SNO.

Reply:
So for OVS offload on secondary network we would have it in nodepolicy ? 
(ovn is not expected to be used in this case)

Reply:
@adrianchris could you clarify the use case of ovs offload on secondary network? is it using ovs-cni if not ovn?

Reply:
@adrianchiris As the offload is enabled in OVS, can we say that all the traffic on OVS can be offloaded, no matter what control plane it connects to? If yes, I think we shall change this flag to `EnableOvsOffload`.

Reply:
@pliurh yes, i think it should be `EnableOvsOffload` , the SDN is not relevant here IMO.

@zshi-redhat it may be

Reply:
The configuration is for sriov node only. I expect it will control the ovs configuration (hw-offload, tc-policy). so it will be used only for hw-offload use case. For other cases like switchdev-cni that doesn't require ovs bridge, we rely on config daemon to configure the device.

Reply:
changed the flag name to `EnableOvsOffload`.